docs/reference/modules/remote-clusters.asciidoc

```diff
@@ -258,13 +258,17 @@ separately.
   The time to wait for remote connections to be established when the node
   starts. The default is `30s`.
 
-`node.remote_cluster_client`::
+`remote_cluster_client` <<node-roles,role>>::
 
   By default, any node in the cluster can act as a cross-cluster client and
-  connect to remote clusters. The `node.remote_cluster_client` setting can be
-  set to `false` (defaults to `true`) to prevent certain nodes from connecting
-  to remote clusters. Remote cluster requests must be sent to a node that is
-  allowed to act as a cross-cluster client.
+  connect to remote clusters. The `remote_cluster_client` role can be excluded
+  excluded from the list of <<node-roles,roles> to prevent certain nodes from
```

Comment:
```suggestion
  connect to remote clusters. The `remote_cluster_client` role can be excluded
 from the list of <<node-roles,roles> to prevent certain nodes from
```